src/main/java/gui/CounterGridPane.java

```diff
@@ -0,0 +1,52 @@
+package gui;
```

Comment:
The class will represent the small sidebar showing counter numbers. Basically this http://puu.sh/oGcW2/c7c957ff6d.jpg
So it has nothing to do with the camera. I'll add comments to better explain its function.